src/api/1/controller-admin/load-securities/index.md

```diff
@@ -0,0 +1,91 @@
+---
+layout: full.html.hbs
+title: loadSecurities
+---
+
+# loadSecurities
+
+{{{since "1.6.6"}}}
+
+Load roles, profiles and users into the storage layer.
+
+The roles, profiles and users definition follow the same structure as in the body parameter of the API:
+
+ - [createRole]({{ site_base_path }}api/1/controller-security/create-role)
+ - [createProfile]({{ site_base_path }}api/1/controller-security/create-profile)
+ - [createUser]({{ site_base_path }}api/1/controller-security/create-user)
+
+**Notes:**
+
+* The file can contain any number of roles, profiles and users.
+* If a role, profile or user already exists, it will be replaced.
```

Comment:
Are currently connected users affected?

Reply:
Yes, they have to reconnected to Kuzzle because their JWT become invalid